accessibility-checker-extension/src/ts/options/OptionsApp.tsx

```diff
@@ -338,12 +338,19 @@ export class OptionsApp extends React.Component<{}, OptionsAppState> {
             <aside aria-label="About Accessibility Checker Options">
             <p>
                 By default, the Accessibility Checker uses a set of rules that correspond to 
-                the most recent WCAG standards plus some additional IBM requirements. Rule sets 
-                for specific WCAG versions are also available. The rule sets are updated regularly, 
+                the WCAG standards plus some additional IBM requirements. Rule sets 
+                are available for specific WCAG versions. The rule sets are updated regularly, 
                 and each update has a date of deployment. If you need to replicate an earlier test, 
-                choose the deployment date of the original test.
+                choose the deployment date of the original test. 
             </p>
-            <p>
+            <p>    
+                Experiment with a preview of a possible future rule set,
+                such as the new WCAG 2.2 related rules, by selecting the "Preview Rules" rule set.
+            </p>
```

Comment:
```suggestion
```
Most of the time, we don't update the extension unless there's a rule update, so "preview" isn't meaningful enough to call out specifically. The WCAG 2.2 Guideline will be available in the latest rules - they do not need to use preview.